src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  align-items: center;
+  max-height: 60px;
+  max-width: 1200px;
+}
+
+.img {
+  display: flex;
+  height: 40px;
+  width: 40px;
+  padding: 10px 0 10px 50px;
```

Comment:
the left padding should be removed from here and added to the header. Also add right padding as well.